ports/raspberrypi/boards/lilygo_t_display_rp2040/mpconfigboard.mk

```diff
@@ -0,0 +1,11 @@
+USB_VID = 0x303A
+USB_PID = 0x8008
```

Comment:
For the record:

A brand new device shows

```bash
$ lsusb
[...]
Bus 003 Device 104: ID 2e8a:000a Raspberry Pi PicoArduino
```

Boot mode

```bash
$ lsusb
[...]
Bus 003 Device 103: ID 2e8a:0003 Raspberry Pi RP2 Boot
```

Reply:
I will try to give it a go to request the PID from raspberry.

Reply:
I submitted the form mentioned on https://github.com/raspberrypi/usb-pid but not sure if it went through (aka. didn't get a conformation).

Reply:
I did the same, no confirmation though. Let's see. Also in the mean time opened an issue on their official github repo to see if we got any luck.

Reply:
Here is the issue. https://github.com/Xinyuan-LilyGO/LILYGO-T-display-RP2040/issues/5

Reply:
Would you suggest I change the value in mpconfigboard.mk to match the lsusb results? I don't think the esp32s2 one is using the factory value though.

Reply:
I would suggest to use **USB_VID = "0x2E8A"** (that's the Raspberry Pi foundation) and **USB_PID = "0x000B"** (that's Circuitpython firmware - according to https://github.com/raspberrypi/usb-pid ). On this page it states in the second sentence after "Use of the IDs" that "_In general, the only reason to require one is because Windows uses the product ID to select a vendor-specific driver for a USB device._" And from USB's perspective it is just connected to a rp2040, and gets some subfunctions like HID, usbstorage and serial. The pin definitions, possible leds and buttons and displays are all part of the circuitpython firmware, not the USB connection.

Further rationale:
Even the micro:bit just uses VID 0x0D28 (NXP) and PID 0x0204 (Arm Embed) for their connection. Actually, using 0x303A would not be correct, that is the VID for Expressif, but the MCU is not an esp32 but rp2040.

Interestingly, the standard rp2040 board is after the flash identified as VID 0x239A (Adafruit) and **not** as a Raspberry Pi Pico (0x2E8A) with circuitpython firmware (PID 0x000B)